### PR TITLE
Update Terra export guide (closes #583)

### LINKED ISSUE #583: Update instructions in "Exporting HCA Data to Terra" guide
- Terra (or Dockstore) bug: When searching for a workflow in Terra the "wdl" in "https://dockstore.org/search?_type=workflow&descriptorType=wdl&searchMode=files" needs to be uppercase "WDL" or Dockstore needs to become case-insensitive on the search.
  - Select "Process multiple workflows" then "Select Data", "Choose specific rows to process", select one file and click "OK".
  - In the Attribute column select the drs URL (e.g. this.\_\_bam\_\_drs\_url).

### PR DESCRIPTION
**Pull request by natanlao**
Closes #583

### CODE REVIEW

File: content/guides/userguides/terra-export.md [hunk 1]
```diff
@@ -105,16 +105,18 @@ Step three: running the workflow in Terra
...
-see a blue *Run Analysis* button pop up. Click that one, and confirm your input
-when prompted. Terra is now running the workflow - take a break for a few minutes,
-grab a coffee, stretch. You deserve it.
+and find the MD5 checksum of that file. Make sure that the *Process multiple
+workflows* radio button is selected, then choose a single file to process by
+navigating to *Select Date* > *Choose specific rows to process*.
+
+Next, tell the workflow how to find the file you selected by setting the
+*inputFile* variable. Click on the *Attribute* field (red box in the
+screenshot above).
```
Comment (chmreid) [inline content/guides/userguides/terra-export.md:114]:
```suggestion
screenshot above).

```

**Review by chmreid — APPROVED** [review]
LGTM!

**Comment by NoopDog** [comment]
HI @chmreid can you revert this please and make this a PR against staging and let me know that it is ok to merge? Thanks Charles. I owe the group a SOP for making PRs. For now please just make a PR against staging for any content and let me know when the content is approved so I can review the markdown etc and merge. 

Actually dont worry about reverting dev, I will force push staging down to dev once this PR is merged to staging

**Comment by chmreid** [comment]
> t me know when the content is approved so I can review the markdown etc and merge.

Done! Thanks for doing the extra housekeeping.

**Pull request merged**